airbyte-webapp/src/packages/cloud/views/auth/OAuthLogin/OAuthLogin.tsx

```diff
@@ -0,0 +1,118 @@
+import { useRef, useState } from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+import { useUnmount } from "react-use";
+import { Subscription } from "rxjs";
+
+import { Spinner } from "components";
+
+import { useExperiment } from "hooks/services/Experiment";
+import { useAuthService } from "packages/cloud/services/auth/AuthService";
+
+import githubLogo from "./assets/github-logo.svg";
+import googleLogo from "./assets/google-logo.svg";
+import styles from "./OAuthLogin.module.scss";
+
+const GitHubButton: React.FC<{ onClick: () => void }> = ({ onClick }) => {
+  return (
+    <button className={styles.github} onClick={onClick} data-testid="githubOauthLogin">
+      <img src={githubLogo} alt="" />
+      <FormattedMessage id="login.oauth.github" tagName="span" />
+    </button>
+  );
+};
+
+const GoogleButton: React.FC<{ onClick: () => void }> = ({ onClick }) => {
+  return (
+    <button className={styles.google} onClick={onClick} data-testid="googleOauthLogin">
+      <img src={googleLogo} alt="" />
```

Comment:
Empty alt?

Reply:
Yes, this image is purely decoratively (the logo on the button), and not fulfilling andy other purpose, thus the correct way to mark it as such for accessibility purposes is to give it an empty `alt` attribute.